### DISCUSSION TITLE
PUBLIC_API not working?

### DISCUSSION THREAD

**Discussion by korneliusz-w** [directus/directus#3369] [Archive]
Hi

How to use PUBLIC_API env variable?

I want to have:
http://cms-directus:8055/admin/  - as normal admin panel URL
http://cms-directus:8055/api/  - as all api links   api/items,  api/collections ....

I set direcuts on apache virtual host

httpd-vhost.conf
```
<VirtualHost localhost:8055>
    ServerName cms-directus
    ProxyPass / http://localhost:8055
</VirtualHost>
```

C:\Windows\System32\drivers\etc\hosts
`127.0.0.1 cms-directus`

directus env
```
PORT=8055
PUBLIC_URL="/api"
```

but it is not correct

http://cms-directus:8055/admin    - not working 
GET http://cms-directus:8055/api/admin/css/app.d287af71.css net::ERR_ABORTED 404 (Not Found)

http://cms-directus:8055/api/collections   - {"errors":[{"message":"Route /api/collections doesn't exist.","extensions":{"code":"ROUTE_NOT_FOUND"}}]}

**Comment by rijkvanzanten** [comment]
There is no `PUBLIC_API` environment variable. If you're talking about `PUBLIC_URL`, it's the base path of where your directus installation can be accessed. It's used for links in emails, static assets, etc. It does not control a custom name for admin vs api or anything like that

**Reply by korneliusz-w** [reply]
OK
